plugins/block-sharable-procedures/src/events_procedure_parameter_create.ts

```diff
@@ -49,23 +49,14 @@ export class ProcedureParameterCreate extends ProcedureParameterBase {
     }
     const parameterModel = procedureModel.getParameter(this.index);
     if (forward) {
-      if (this.parameterMatches(parameterModel)) return;
+      if (parameterModel && this.parameterMatches(parameterModel)) return;
```

Comment:
Optional: I think it might be a bit nicer to make the arg to parameterMatches nullable, and do the non-null check in there, returning false if it is null?

Reply:
Ahhh ok I misunderstood what you were saying before. That makes much more sense haha. Sounds good!